misc/HelloWorldWithWebRoot/src/HelloWorldWithWebRoot/Microsoft.AspNet.Hosting.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Hosting": {
+        "WebRoot": "wwwroot"
```

Comment:
Let's make this look for just "WebRoot" withing the config system first, and then only if there's no value, try looking in "Hosting:WebRoot". Then we can just make this file:

``` JSON
{
    "WebRoot": "wwwroot"
}
```